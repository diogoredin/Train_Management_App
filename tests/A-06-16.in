6
3
1
Lisboa - Oriente
Pragal
2017-12-25
05:50
0
0
0
