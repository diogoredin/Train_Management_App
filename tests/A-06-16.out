Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2017-12-25 @ 16.04
Serviço #590 @ 9.64
07:02 Lisboa - Oriente
07:10 Lisboa - Entrecampos
07:57 Poceirão
08:06 Pegoes
08:10 São João das Craveiras
08:21 Vendas Novas
Serviço #690 @ 6.40
08:31 Vendas Novas
08:53 Pinhal Novo
09:14 Pragal
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 